transform_inline.ml

```diff
@@ -56,10 +57,13 @@ let inline ({main; functions} as orig_prog : program) : program option =
      accordingly *)
   let replace_vars caller ({formals; instrs} as callee) =
     let caller_vars = function_vars caller in
-    let callee_vars = function_vars callee in
+    let callee_vars = VarSet.elements (function_vars callee) in
     let replacements =
-      List.map (fun old -> (old, fresh caller_vars old))
-        (VarSet.elements callee_vars)
+      let replace (used , acc) old =
+        let fresh = fresh used old in
```

Comment:
I think it would be clearer to name it `fresh_var`. (Yeah, I don't like shadowing/reusing names.)